cypress/e2e/pages/onDemandAudio/index.cy.js

```diff
@@ -1,32 +1,487 @@
+    tests,
+  },
+  {
+    path: '/burmese/podcasts/p02pc9lh', // Podcast brand
+    service: 'burmese',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/burmese/podcasts/p02pc9lh/p09kzply', // Podcast Episode
+    service: 'burmese',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/gahuza/bbc_gahuza_radio/programmes/p0340x2m', // On Demand Brand
+    service: 'gahuza',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/gahuza/bbc_gahuza_radio/w3ct1v5v', // On Demand Episode
+    service: 'gahuza',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/gahuza/podcasts/p07yh8hb', // Podcast Brand
+    service: 'gahuza',
+    runforEnv: ['local', 'test', 'live'],
+    tests,
+  },
+  {
+    path: '/gahuza/podcasts/p07yh8hb/p09km4t4', // Podcast Episode
+    service: 'gahuza',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/hausa/bbc_hausa_radio/programmes/p030s4mh', // On Demand Brand
+    service: 'hausa',
+    runforEnv: ['live'],
+    tests,
+  },
+  {
+    path: '/hausa/bbc_hausa_radio/programmes/p030s4mx', // On Demand Brand
+    service: 'hausa',
+    runforEnv: ['test'],
+    tests,
+  },
+  {
+    path: '/hausa/bbc_hausa_radio/w3ct1qzn', // On Demand Episode
+    service: 'hausa',
+    runforEnv: ['live'],
+    tests,
+  },
+  {
+    path: '/hausa/bbc_hausa_radio/w3ct1033', // On Demand Episode
+    service: 'hausa',
+    runforEnv: ['test'],
+    tests,
+  },
+  {
+    path: '/hausa/podcasts/p08mlgcb', // Podcast Brand
+    service: 'hausa',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/hausa/podcasts/p08mlgcb/p09l0fw6', // Podcast Episode
+    service: 'hausa',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/hindi/podcasts/p0552909', // Podcast Brand
+    service: 'hindi',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/hindi/podcasts/p0552909/p09kjqjm', // Podcast Episode
+    service: 'hindi',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/indonesia/bbc_indonesian_radio/programmes/w13xtt0s', // On Demand Brand
+    service: 'indonesia',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/indonesia/bbc_indonesian_radio/w172xybnvm6718v', // On Demand Episode
+    service: 'indonesia',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/indonesia/podcasts/p02pc9v6', // Podcast Brand
+    service: 'indonesia',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/indonesia/podcasts/p02pc9v6/p09l1mhb', // Podcast Episode
+    service: 'indonesia',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/korean/bbc_korean_radio/programmes/w13xttll', // On Demand Brand
+    service: 'korean',
+    runforEnv: 'live',
+    tests,
+  },
+  {
+    path: '/korean/bbc_korean_radio/programmes/w13xttlm', // On Demand Brand
+    service: 'korean',
+    runforEnv: 'test',
+    tests,
+  },
```

Comment:
Because test points to live data, we can pick one path and combine them into a single entry, with `runforEnv: ['test', 'live']`